ee/modules/140-user-authz/templates/multitenancy/role-bindings.yaml

```diff
@@ -0,0 +1,73 @@
+{{- range $crd := .Values.userAuthz.internal.multitenancyCRDs }}
+  {{- range $ns := $crd.spec.limitNamespaces }}
+  {{- if hasKey $crd.spec "accessLevel" }}
+    {{- if not (list "User" "PrivilegedUser" "Editor" "Admin" "ClusterEditor" "ClusterAdmin" "SuperAdmin" | has $crd.spec.accessLevel) }}
+      {{- cat "Unsupported accessLevel type" $crd.spec.accessLevel "in" $crd.name "crd" | fail }}
+    {{- end }}
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: user-authz:{{ $crd.name }}:{{ $crd.spec.accessLevel | kebabcase }}
```

Comment:
it is better to add d8: prefix!

Reply:
Well, current module [doesn't contain](https://github.com/deckhouse/deckhouse/blob/main/modules/140-user-authz/templates/cluster-role-bindings.yaml#L8) this prefix. Are you sure, that we have to rename all of this?